src/app/app.component.js

```diff
@@ -30,15 +32,40 @@ function AppComponentCtrl(
 
     this.settings = dimSettingsService;
     $scope.$watch(() => this.settings.itemSize, (size) => {
-      document.querySelector('html').style.setProperty("--item-size", `${size}px`);
+      if (!this.isPhonePortrait) {
```

Comment:
./mediaQueries also has a function to test isPhonePortrait at any time so you don't necessarily have to subscribe to it and set it on the controller.

Reply:
I did that way because I was having problems without it, but I could be missing something else that I solved implementing this way.
I'll try again without subscribing.